.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+Contributing to Goss
+====================
+
+Thank you for your interest in contributing to Goss. Goss wouldn't be where it is today if it wasn't for people like you. Some ways you can contribute:
+
+* Improve the [README](https://github.com/aelsabbahy/goss/blob/master/README.md) and/or [Docs](https://github.com/aelsabbahy/goss/blob/master/docs/manual.md) - This makes it easier for new users to learn goss.
+* Vote on bugs and feature requests by adding a thrumbs up to the inital post.
+* Create tutorials, blog posts and example use-cases on how to use Goss.
+* Help users with [questions](https://github.com/aelsabbahy/goss/labels/question) tracker.
+* Fix verified [bugs](https://github.com/aelsabbahy/goss/labels/bug).
```

Comment:
Perhaps we could add a `triaged` label so it's possible for contributors to see the list of _verified_ bugs/FRs vs ones that have not yet been verified/accepted by the maintainer(s)?